CWE/events/Essential events 3.txt

```diff
@@ -2087,116 +2087,6 @@ country_event = {
 	}
 }
 
-#Pietro Badoglio War Trials
```

Comment:
Please do not remove these events as they are still worthwhile ones to include.

Reply:
I will change the effects then since as of now they can break gameplay either by making communism too strong or too weak

Reply:
Thanks